src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js

```diff
@@ -1,39 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
+import { Button, themeSpacing, themeColor } from '@datapunt/asc-ui';
+import { ChevronRight, ChevronLeft } from '@datapunt/asc-assets';
 import { attachmentsType } from 'shared/types';
 
-import './style.scss';
+const IconButton = styled(Button).attrs({
+  size: 32,
+  iconSize: 16,
+  variant: 'application',
+})``;
+
+const PreviousButton = styled(IconButton)`
+  position: absolute;
+  left: 10px;
+  top: calc(50% - ${themeSpacing(4)});
+`;
+
+const NextButton = styled(IconButton)`
+  position: absolute;
+  right: 10px;
```

Comment:
themeSpacing?

Reply:
The button doesn't have to align to other elements, so theme spacing values here aren't necessary.